ui/components/src/TimeSeriesTooltip/focused-series.test.ts

```diff
@@ -62,7 +62,7 @@ describe('getNearbySeries', () => {
       kind: 'Decimal',
       decimal_places: 2,
     };
-    expect(getNearbySeries(chartData, pointInGrid, yBuffer, decimalUnit)).toEqual(focusedSeriesOutput);
+    expect(getNearbySeries(chartData, pointInGrid, yBuffer, undefined, decimalUnit)).toEqual(focusedSeriesOutput);
```

Comment:
could be nice to name these someday, but not needed right now

i.e. `getNearbySeries({ chartData, pointInGrid, yBuffer, chart: undefined, unit: decimalUnit})`

Reply:
+1 on this as a future thing. Once the number of args gets beyond a small handful, it's much easier to work with an object.

Reply:
Yeah agreed! I'll probably wait on this though, Matt is going to be bringing in annotation rendering changes, which also adjusts this function (they render events as 'scatter' series so a condition is added to show different tooltip content). 

Avoiding further changing the params for now may make his life a bit easier since there are going to be a lot of merge conflicts already!